django/db/models/deletion.py

```diff
@@ -107,6 +107,8 @@ def __init__(self, using, origin=None):
         # fast_deletes is a list of queryset-likes that can be deleted without
         # fetching the objects into memory.
         self.fast_deletes = []
+        # set perform_signals to false in order to ignore signals
```

Comment:
I think this is sufficiently clear from the name `perform_signals` that we don't need this comment.